site/map.js

```diff
@@ -0,0 +1,365 @@
+/* globals mapboxgl */
+
+// import * as mapboxgl from 'mapbox-gl/dist/mapbox-gl.js';
```

Comment:
?
is it coming in as a global now? just delete this?

Reply:
I was in the middle of getting it to come from a module, but it fought back, I'll leave it in since the next thing i'm gonna do is uncomment this and start fighting it again.